models/station.py

```diff
@@ -0,0 +1,43 @@
+import sqlalchemy as sa
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship, sessionmaker
+
+import config.db_config as db_config
+
+Base = declarative_base()
+
+db = f'postgresql://{db_config.POSTGRES_USER}:{db_config.POSTGRES_PASSWORD}@localhost:{db_config.POSTGRES_OUT_PORT}/postgres'
```

Comment:
The "models" folder should store only models. Move the initialization to a service folder.